FisherYatesShuffle/Python/FisherYatesShuffle.py

```diff
@@ -0,0 +1,29 @@
+import random
+
+
+def FischerYatesShuffle(arr):
+    """
+    Shuffle an array using Fischer Yates algorithm
+    [https://en.wikipedia.org/wiki/Fisher-Yates_shuffle]
+
+    :param arr:
+    :return: shuffled array
+    """
+
+    for i in range(0, len(arr)):
+        j = random.randrange(0, i + 1)
+        tmp = arr[i]
+        arr[i] = arr[j]
+        arr[j] = tmp
+
+    return arr
+
+
+testArr = [i ** 2 for i in range(20)]
```

Comment:
Put in the test code into a ```main``` function. And call it. See [here](https://docs.python.org/2/library/__main__.html)